doc/developer-guide.md

```diff
@@ -755,6 +755,7 @@ cmake . -GNinja \
   -DLLVM_HOST_TRIPLE=arm-unknown-linux-gnu \
```

Comment:
Possibly worth adding to the changelog, but feel free to ignore.

Reply:
Good to think about, but in `CHANGELOG.md`, we document the things that users need to take into account when switching to a newer version. This isn't like that, users where zstd was not detected aren't affected and can continue to not specify `LLVM_ENABLE_ZSTD`, users where zstd was detected would already have had to work around it by manually specifying `LLVM_ENABLE_ZSTD=OFF` (or `FALSE` or otherwise false) so this tells them they would already have known and don't need to specifically take into account again now. If you're okay with leaving it out, I'll leave it out :)